src/query/shorthand.ts

```diff
@@ -53,7 +56,13 @@ export function spec(specQ: SpecQuery,
     parts.push(value(specQ.mark, replace[Property.MARK]));
   }
 
-  // TODO: transform
+  if (specQ.transform) {
+    const calc = stringifyCalculate(specQ.transform.calculate);
+    const filter = stringifyFilter(specQ.transform.filter);
+
+    parts.push((calc ? `calculate:${calc}` : '') +
```

Comment:
Plus, don't push empty string if a particular part is empty.  Otherwise you get `||`